rfd/0034-desktop-access-windows.md

```diff
@@ -0,0 +1,256 @@
+---
+authors: Andrew Lytvynov (andrew@goteleport.com)
+state: draft
+---
+
+# RFD 34 - Desktop Access - Windows
+
+## What
+
+RFD 33 defines the high-level goals and architecture for Teleport Desktop
+Access.
+
+This RFD specifies how Teleport Desktop Access integrates with Windows hosts,
+including Microsoft Active Directory domains.
+
+## Details
+
+### Architecture
+
+Windows Desktop Access is implemented using `windows_desktop_service` that
+translates the Teleport desktop protocol into RDP:
+
+```
++--------+
+| web UI |
++--------+
+    ^
+    | desktop protocol over websocket
+    v
++--------+
+| proxy  |
++--------+
+    ^
+    | desktop protocol over mTLS
+    v
++-------------------------+
+| windows_backend_service |--------------\
++-------------------------+-\            |
+   ^                        |            |
+   | RDP                    | RDP        | LDAP
+   v                        v            |
++---------------------+  +-----------+  +-------------------+
+| windows server 2012 |  | windows 7 |  | domain controller |
++---------------------+  +-----------+  +-------------------+
+```
+
+`windows_desktop_service` can talk to any number of remote Windows RDP hosts.
+It can also talk to `localhost` RDP service, if installed on a Windows machine
+in agent mode (described below).
+
+`windows_desktop_service` also discovers all available Windows hosts from
```

Comment:
I see below under host discovery that this is not the only option and a hard-coded list of hosts is also supported. It might help to clarify that here. On first read I took this to mean that all hosts on the domain would be automatically exposed to teleport (which may or may not be desirable)

Reply:
Clarified in https://github.com/gravitational/teleport/pull/7841, please review